src/diffusers/pipelines/alt_diffusion/modeling_roberta_series.py

```diff
@@ -65,6 +67,8 @@ def __init__(self, config):
         super().__init__(config)
         self.roberta = XLMRobertaModel(config)
         self.transformation = nn.Linear(config.hidden_size, config.project_dim)
+        self.transformation_pre = nn.Linear(config.hidden_size, config.project_dim)
```

Comment:
We should only add those under the condition that it's a m18 checkpoint otherwise this breaks for previous checkpoints. Could we add a `config.has_pre_transformation` flag?

Reply:
> We should only add those under the condition that it's a m18 checkpoint otherwise this breaks for previous checkpoints. Could we add a `config.has_pre_transformation` flag?

Exactly! I also think so, but I don't know where to add this flag properly, so I am looking for your help.